src/olympia/amo/tests/__init__.py

```diff
@@ -726,9 +726,8 @@ def req_factory_factory(url, user=None, post=False, data=None, session=None):
 def user_factory(**kw):
     global user_factory_counter
     username = kw.pop('username', u'factoryûser%d' % user_factory_counter)
-
-    user = UserProfile.objects.create(
-        username=username, email='%s@mozilla.com' % username, **kw)
+    email = kw.pop('email', 'factoryuser%d@mozilla.com' % user_factory_counter)
```

Comment:
There was, and I had shaved enough yaks for this change... I'm not entirely sure what we should do with unicode emails...